core/src/impl/Kokkos_Profiling.hpp

```diff
@@ -71,7 +71,7 @@ void startSection(const uint32_t secID);
 void stopSection(const uint32_t secID);
 void destroyProfileSection(const uint32_t secID);
 
-void markEvent(const std::string* evName);
+void markEvent(const std::string& evName);
```

Comment:
Didn't we already merge that in?

Reply:
We apparently did. I must be missing something with git here (I'm not great with it). I thought this page showed a diff from `develop`, which makes me confused why I see the left hand side still having the pointer formulation. Basically I've merged the PR I used to make that fix into my branch, I'm not sure why this doesn't just show that both branches have the fix

Reply:
Yeah, I have the same commit in this branch as the one that merged into develop. I have no idea what's up there

Reply:
Jeremy helped me out, that's a redundant commit, something I merged into my branch since I branched off of develop to implement this.